code/modules/power/apc.dm

```diff
@@ -828,7 +831,7 @@
 	else if(stat & (BROKEN|MAINT))
 		to_chat(user, span_warning("Nothing happens!"))
 	else
-		if(allowed(usr) && !wires.is_cut(WIRE_IDSCAN) && !malfhack)
+		if(allowed(usr) && !wires.is_cut(WIRE_IDSCAN) && !malfhack && !integration_cog)
```

Comment:
does thi s work

Reply:
no, no it does not, i misunderstood what those were trying to do, they're preventing people from locking or unlocking, not letting people bypass allowed